week9/answers.sql

```diff
@@ -1,3 +1,129 @@
 -- Create your tables, views, functions and procedures here!
 CREATE SCHEMA social;
 USE social;
+
+-- Create Users Table
+CREATE TABLE users (
+  user_id INT PRIMARY KEY,
```

Comment:
Throughout your tables, the following is missing:

- AUTO_INCREMENT
- NOT NULL